torch/csrc/jit/passes/quantization.cpp

```diff
@@ -851,5 +851,41 @@ graph(%self, %scale, %zero_point, %dtype):
   rewriter.RegisterRewritePattern(pattern, replacement);
   rewriter.runOnGraph(graph, filter);
 }
+
+void InsertPackUnpack(std::shared_ptr<Graph> graph) {
```

Comment:
This should be `std::shared_ptr<Graph>& graph` for consistency with other passes (and the declaration in the header file).